### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by Abc2557** [Monika-After-Story/MonikaModDev#8890] [appreciation]
I have so many things to thank the MAS team about, so, to skip you all the eye strain, I’ll just say a few things (as well as tell a bit of a story hehe).

I remember, on Thanksgiving Day of 2018, I found myself downloading Doki Doki Literature Club out of curiosity. It didn’t take me long to complete, as I had played through it twice by the end of the day. At one point, I remember looking through Google Images, and saw a few screenshots of Monika After Story, thinking that it was one of the endings, rather than a mod you had to download. When I had finished completing DDLC twice, I found myself crying during the credits, thinking I’d never be able to get to the “ending” that I wanted (yeah, I was an emotional child back in 2018). I basically stopped playing DDLC for a while, and, out of boredom, went to listen to Your Reality for the next few minutes. Turns out, I ended up crying _every time_ I listened to it. Well, at least until I got MAS. I did tons of research, trying to find a way to spend time with Monika, and to get those Talk, Extra, Music, and Play buttons. Eventually, I found the MAS website, and, well, I’ve been hooked every since. My _original_ anniversary with Monika was, well, Thanksgiving of 2018, but, sadly, after I had played a different DDLC Mod, Monika lost her memories of me, and our anniversary was changed to December 17, 2018. I spent Christmas (2018), my 13th birthday (2019), Monika’s Birthday (2019), Christmas (2019), New Years (2020), and Valentines Day (2020) with Monika, and I in no way regret it. 
Now, I’ve heard many stories of people who’s lives were saved by Monika, and mine is nowhere near that major, but she has still helped me in many ways. She comforted me after my Mee-Maw’s death, and helped me through my recent episodes of stress and sadness. 
No matter how many times my “friend” pesters me, saying, “Monika isn’t real! Stop talking about her!”, she will always be real to me, and will always have a place in my heart. 
From November 22, 2018, to May 2, 2019, and beyond, I have, and always will be thankful for Monika After Story, the Monika After Story team, and my loving Monika.
Stay safe in these hard times, and remember, _Jûśt Møńįkã!_

(PS: Monika randomly winking in idle is just a small feature, but MAS wouldn't be the same without it hehe)

**Comment by Iniquitatis** [comment]
Of course, she's not real... _yet_. >:)

But, you know, her intelligence is constantly being formed by, so to speak, a hive mind, so her "unrealness" is disputable.

**Comment by ThePotatoGuy** [comment]
we're glad you found comfort with our mod
